src/app/shell/personal-cabinet/provider/provider-org-info/provider-org-info.component.html

```diff
@@ -29,7 +29,7 @@ <h5>Телефон</h5>
           <p>{{provider.phoneNumber}}</p>
           <h5>Емейл</h5>
           <p>{{provider.email}}</p>
-          <h5>Вебсайт</h5>
+          <h5 *ngIf='provider.website'>Вебсайт</h5>
```

Comment:
can you add please the same functionality and display Instagram,Facebook fileds? 
![image](https://user-images.githubusercontent.com/27493959/151793181-8d3be513-fd16-4a60-aed0-32872d750858.png)

Reply:
Ok, added for Instagram and fb